remote-application-jasmine.setup.ts

```diff
@@ -0,0 +1,28 @@
+import {SpecReporter, StacktraceOption} from 'jasmine-spec-reporter';
+import {StandaloneTestCaseReporter} from "./src";
```

Comment:
As this file is just for the sake of testing WDIO remote, I think it should be placed under the test/remote folder

Reply:
Moved